roles/ealapps/tasks/main.yml

```diff
@@ -25,10 +10,26 @@
     mode: 0775
   changed_when: false
 
+- name: ealapps | create directories for shared files
+  ansible.builtin.file:
+    path: '{{ capistrano_base_dir }}/{{ item }}'
+    state: 'directory'
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+    mode: 0755
+  changed_when: false
+  with_items:
```

Comment:
can we make this use the preferred [loops](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html)